### PR TITLE
Fixes #2333: Add systemdb export metadata procedure

### LINKED ISSUE #2333: Add systemdb export metadata procedure
Add procedure `apoc.systemdb.export.metadata()` 
to export features present systemdb apoc nodes (custom procedures/function, uuid, triggers...),
in order to import them in another database (for example after a `./neo4j-admin restore`)

### PR DESCRIPTION
**Pull request by vga91**
Fixes #2333

I kept some of `DocsTest.java` generated files to link them from `database-introspection/systemdb.adoc`, because `overview/apoc.systemdb/apoc.systemdb.export.metadata.adoc` (and files included in this one) contain the signature, usage example and configuration description.

### CODE REVIEW

File: core/src/main/java/apoc/util/Util.java [hunk 1]
```diff
@@ -932,4 +935,14 @@ public static String encodeUserColonPassToBase64(String userPass) {
     public static boolean isSelfRel(Relationship rel) {
         return rel.getStartNodeId() == rel.getEndNodeId();
     }
+
+    public static String toCypherMap(Map<String, Object> map) {
```
Comment (conker84) [inline core/src/main/java/apoc/util/Util.java:939]: Use `CypherFormatterUtils.formatProperties`

File: full/src/main/java/apoc/systemdb/SystemDb.java [hunk 2]
```diff
@@ -48,6 +63,41 @@ public NodesAndRelationshipsResult(List<Node> nodes, List<Relationship> relation
         }
     }
 
+    @Procedure(name = "apoc.systemdb.export.metadata")
```
Comment (conker84) [inline full/src/main/java/apoc/systemdb/SystemDb.java:66]: `@Admin`

Comment (vga91) [reply full/src/main/java/apoc/systemdb/SystemDb.java:68]: Added

File: core/src/main/java/apoc/util/Util.java (hunk 1 above)
Comment (vga91) [reply core/src/main/java/apoc/util/Util.java:941]: Updated

**Comment by conker84** [comment]
Thank you @vga91

**Pull request merged**